### PR TITLE
Branch addition

### PR DESCRIPTION
**Pull request by PoobalanAatmikaLakshmi**

### CODE REVIEW

File: docs/uml-diagrams/DeleteHelpCommand-DeleteHelpCommand.png
Comment (honglinshang) [inline docs/uml-diagrams/DeleteHelpCommand-DeleteHelpCommand.png:1]: i think in this case, you should use `DeleteHelpCommand` directly, like there isn't any `HelpCommand` "created", like how all commands are technically `Command`s but we don't specify a separate `Command` creation bc its technically the same object
ie. `Parser -> DeleteHelpComand ** : chooseHelpCommand("delete")`

File: docs/uml-diagrams/Storage.png
Comment (honglinshang) [inline docs/uml-diagrams/Storage.png:1]: the arrow from `Storage` to `LoggerInterface` shld be a dotted one w blank arrowhead (`..|>`) if i'm not wrong

Comment (honglinshang) [inline docs/uml-diagrams/Storage.png:1]: may want to include {abstract} for Task's title/name thing

**Review by honglinshang — COMMENTED** [review]
Good job correcting the previous changes!

**Review by yixuann02 — COMMENTED** [review]
Good job amending your diagrams, will merge for you

**Review by yixuann02 — APPROVED** [review]

**Pull request merged**